apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx

```diff
@@ -43,6 +44,10 @@ import {
   useUserV3SwapEnable,
   useRoutingSettingChanged,
 } from 'state/user/smartRouter'
+import { useAllowNotifications } from 'state/notifications/hooks'
+import { useManageSubscription, useW3iAccount } from '@web3inbox/widget-react'
+import { Events } from 'views/Notifications/constants'
+import { errorBuilder } from 'views/Notifications/utils/errorBuilder'
```

Comment:
no import from `views` inside `components`. I may cause dependency loop.

'->' stands for "relies on"

✅: views -> components
❌: components -> views